config/database.yml

```diff
@@ -1,18 +1,30 @@
-
 default: &default
-  adapter: postgresql
-  encoding: unicode
+  adapter: sqlite3
```

Comment:
- Please make sure your default database adapter uses `postgresql`
```ruby
default: &default
adapter: postgresql
encoding: unicode

```

Reply:
@rotimiazeez I don't understand why I need to put it in default, as you can see I have specified postgresql for development mode and production mode, which means that I'm using the Postgres database. I just used SQLite for test, and because of it I put it as default. I'm using this way, which is doing the job. Please explain to me why do I need to put it as default?